src/main/java/com/example/application/ui/horizontal/community/IdeasManagementView.java

```diff
@@ -30,7 +30,7 @@
  *  @author Litharshiga Sivarasa, Jessica Reistel, Vanessa Skowronsky
  *  @version 3.0
  *  @since 15.12.2020
- *  @lastUpdated 07.02.2021 by Jessica Reistel
+ *  @lastUpdated 10.02.2021 by Jessica Reistel
```

Comment:
When adding only comments you don't need to change the lastUpdated :') just leave it as it is